type: [feat], message: Adding project structure and some utility packages. (#1)
NeowayLabs/semantic-release: .github/pull_request_template.md

## What?

Here goes your feature/bug-fix proposal

### Type of change?

- [ ] [build]: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
- [ ] [ci]: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
- [ ] [docs]: Documentation only changes
- [ ] [feat]: A new feature
- [ ] [fix]: A bug fix
- [ ] [perf]: A code change that improves performance
- [ ] [refactor]: A code change that neither fixes a bug nor adds a feature
- [ ] [style]: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
- [ ] [test]: Adding missing tests or correcting existing tests

Note: Remove the non-checked options

## Why?

> Here you express the reason it is necessary (keep the _e daí?_ factor in mind)

Reviewer: ```suggestion
Here you express the reason it is necessary (keep the _so what?_ factor in mind)
```
@esequielvirtuoso what do you think?

Author: Better than "_e daí?_"!
I'm changing this one.

Reviewer: _e dai_ is the best haha just to keep the English pattern

## How?

Here you give a little more detail on the implementation or justify any unorthodox choice or practice.

## How has this been tested?

- [ ] Unit tests
- [ ] Integration tests
- [ ] Benchmark tests

> # Checklist:

Reviewer: This looks like a development checklist and not a PR one. maybe change the title to something like `before submitting a PR please make sure to:`. what do you think?

Author: Great tip.
I'll do this.
